.github/COMMIT_CONVENTION.md

```diff
@@ -0,0 +1,51 @@
+## Git Commit Message Convention
+
+> This is adapted from [Angular's commit convention](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular).
+
+#### TL;DR:
+
+Messages must be matched by the following regex:
+
+<!-- prettier-ignore -->
+```js
+/^(revert: )?(feat|fix|docs|refactor|perf|test|ci|chore|release)(\(.+\))?: .{1,50}/
+```
+
+#### Examples
+
+Appears under "Features" header:
+
+```
+feat(@material-svelte/button): add a specific new feature
+```
+
+Appears under "Bug Fixes" header:
+
+```
+fix(@material-svelte/button): fix a specific issue
+```
+
+### Type
+
+If the prefix is `feat`, `fix` or `perf`, it will appear in the changelog. However, if there is any [BREAKING CHANGE](#footer), the commit will always appear in the changelog.
```

Comment:
I think there's something missing in this sentence 👍 
edit: or maybe just swapping the sentence with the one below, so "If prefix feat.." "Other prefixes" "Suggested prefixes" "However, if.."